etc/profile-a-l/irssi.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for irssi
+# Description: IRC client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include irssi.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}/wayland-*
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-X11.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.irssi/
+whitelist ${HOME}/.irssi/
+
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+no3d
+nodvd
+nogroups
+nonewprivs
+noprinters
+noroot
+nosound
+notv
+nou2f
+novideo
```

Comment:
Add and sort:

```
noinput
notpm
```

Note: `notpm` is in firejail-git.  When creating a new profile it's recommended
to use the latest development version to ensure that an up-to-date
profile.template is used.

Reply:
I did use the template 4mo ago, when notpm was added to the template :)

Reply:
hum... i thought i had issues with noinput but now i tested and can't find why.  adding it back then